keyboards/keyprez/rhino/readme.md

```diff
@@ -0,0 +1,21 @@
+# rhino
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Christian Sandven](https://github.com/yourusername)
+* Hardware Supported: *Pro micro compatible boards*
+* Hardware Availability: *TBA*
```

Comment:
Please fill these out.

Reply:
@fauxpark I updated the GitHub username now. I currently do not have a website for the board yet, but it is in the works. Can I add the hardware availability in a future branch?